### DISCUSSION TITLE
URLs showing a :80 port

### DISCUSSION THREAD

**Discussion by pietrogregorini** [typemill/typemill#420] [Q&A]
I was trying to install Typemill 2.0.1 on my server online but some issues occurred.
When I try to access the setup page it doesn't load the stylesheet, so I checked the source and noticed that all the URLs are looking for an :80 port – eg: domain.com:80/etc.

I tried editing the .htaccess file, adding the RewriteBase for the directory when I host Typemill, but with no changes.

Any ideas about how can I solve this?
Thanks 🙏

**Comment by trendschau** [comment]
port :80 looks like a server configuration, another idea is to check if mod_rewrite is installed correctly and if htaccess file is processed correctly. I think there are several tickets about that topic, I tagged many of them with the label "setup": https://github.com/typemill/typemill/issues?q=label%3Asetup+

I hope you will find a solution there
